parm/config/gfs/config.ocnanalletkf

```diff
@@ -8,4 +8,11 @@ echo "BEGIN: config.ocnanalletkf"
 # Get task specific resources
 . "${EXPDIR}/config.resources" ocnanalletkf
 
+export MARINE_LETKF_EXEC="${JEDI_BIN}/gdas.x"
+export MARINE_LETKF_YAML_TMPL="${PARMgfs}/gdas/soca/letkf/letkf.yaml.j2"
+export MARINE_LETKF_STAGE_YAML_TMPL="${PARMgfs}/gdas/soca/letkf/letkf_stage.yaml.j2"
```

Comment:
Should these be `OCNANALLETKF` as the configs suggest?

Reply:
`MARINE` because they cover ocean and ice - do the tasks need to be renamed?

Reply:
never mind - we'll change this

Reply:
changed